src/coreclr/gc/gcconfig.h

```diff
@@ -144,6 +144,9 @@ class GCConfigStringHolder
     INT_CONFIG   (GCDynamicAdaptationMode,   "GCDynamicAdaptationMode",   "System.GC.DynamicAdaptationMode",   1,                  "Enable the GC to dynamically adapt to application sizes.")                               \
     INT_CONFIG   (GCDTargetTCP,              "GCDTargetTCP",              "System.GC.DTargetTCP",              0,                  "Specifies the target tcp for DATAS")                                                     \
     INT_CONFIG   (GCDBGCRatio,               "GCDBGCRatio",               NULL,                                0,                  "Specifies the ratio of BGC to NGC2 for HC change")                                       \
+    INT_CONFIG   (GCDGen0GrowthPercent,      "GCDGen0GrowthPercent",      "System.GC.DGen0GrowthPercent",      0,                  "Specifies the percentage of the default growth factor")                                  \
+    INT_CONFIG   (GCDGen0GrowthMinPerMil,    "GCDGen0GrowthMinPerMil",    "System.GC.DGen0GrowthMinPerMil",    0,                  "Specifies the minimum growth factor")                                                    \
```

Comment:
Is this Min growth Multiplier? Can the naming be simplified?

Reply:
this was the simplest name I came with :) it's the same as GCDGen0GrowthPercent except it uses permil.... do you have a suggestion? it's the number we multiply the stable size with (see `compute_total_gen0_budget`).

Reply:
Maybe call it GCDGen0GrowthMinFactor ?

Reply:
Looks good. Thanks